src/java/com/phenix/pct/AnalyticsBuildListener.java

```diff
@@ -0,0 +1,100 @@
+package com.phenix.pct;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.Map.Entry;
+
+import org.apache.tools.ant.BuildEvent;
+import org.apache.tools.ant.BuildListener;
+import org.apache.tools.ant.Task;
+
+public class AnalyticsBuildListener implements BuildListener {
+    private static final Map<String, Integer> taskCount = new HashMap<>();
+
+    private boolean isPCTTask(Task task) {
+        if (task.getTaskType().toLowerCase().startsWith("pct"))
+            return true;
+        if ("DlcVersion".equalsIgnoreCase(task.getTaskType())
+                || "RestGen".equalsIgnoreCase(task.getTaskType())
+                || "ClassDocumentation".equalsIgnoreCase(task.getTaskType())
+                || "ABLUnit".equalsIgnoreCase(task.getTaskType()))
+            return true;
+        return false;
+    }
+
+    @Override
+    public void taskFinished(BuildEvent event) {
+        Task task = event.getTask();
+        if (task == null) {
+            return;
+        }
+        String taskName = task.getTaskType();
+        if (taskName == null) {
+            return;
+        }
+        if (isPCTTask(task)) {
+            Integer i = taskCount.get(taskName.toLowerCase());
+            if (i == null)
+                taskCount.put(taskName.toLowerCase(), 1);
+            else
+                taskCount.put(taskName.toLowerCase(), i + 1);
+        } else {
+            System.out.println("oups");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS106)